packages/tpetra/core/inout/Tpetra_MatrixIO_def.hpp

```diff
@@ -51,11 +51,23 @@
 namespace Tpetra {
 namespace Utils {
 
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
 template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
+TPETRA_DEPRECATED
```

Comment:
Did you mean to deprecate this overload too?  It doesn't take a Node.

Reply:
Thanks, @mhoemmen 
We decided on Tuesday to deprecate generateMatrix, since it is not used nor tested anywhere in Trilinos.
Is that still OK?

Reply:
@kddevin Ah, sorry, forgot about that :-)  Definitely OK!